doc/internal/testing.rst

```diff
@@ -45,26 +45,26 @@ There are currently 3 types of interactive test cases:
 Running tests
 -------------
 
-The pyglet test suite is based on the `py.test framework <http://pytest.org>`_.
```

Comment:
Good change. It seems like the "dot" has been deprecated since pytest 3.0.

Reply:
Yep. It was also originally scheduled for removal in 6.x as far as I remember.